### DISCUSSION TITLE
Can i transfer note combo from an old build?

### DISCUSSION THREAD

**Discussion by Stompyyyy** [FunkinCrew/Funkin#3733] [Q&A]
Ok so i have an old build of FNF, which has the note combo system. Can i transfer it into the new FNF version
If not then why?

**Comment by AbnormalPoof** [comment]
I mean, there's nothing stopping you. The assets are there, all you need to do is re-implement it using HScript.

**Comment by Stompyyyy** [comment]
Ohhhhh so coding is required i see, what folder do i need to edit exactly if you don’t mind?
